src/vcpkg/vcpkgpaths.cpp

```diff
@@ -123,11 +119,11 @@ namespace vcpkg
         auto parsed_config_opt = reader.visit(obj, get_configuration_deserializer());
         if (!reader.errors().empty())
         {
-            print2(Color::error, "Error: while parsing ", config_path, "\n");
+            msg::println_error(msgFailedToParseConfig, msg::path = config_path);
             for (auto&& msg : reader.errors())
-                print2("    ", msg, '\n');
+                msg::write_unlocalized_text_to_stdout(Color::none, fmt::format("    {}\n", msg));
 
-            print2("See ", docs::registries_url, " for more information.\n");
+            msg::println(msgExtendedDocumentationAtUrl, msg::url = docs::registries_url);
             Checks::exit_fail(VCPKG_LINE_INFO);
```

Comment:
Should this whole thing be printed as one error message?

Reply:
msgExtendedDocumentationAtUrl is used in other places. I could join them, but it wouldn't reduce the number of localized messages.